tools/scripts/ci_lint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+  paths=$(git diff --diff-filter ACMR --name-only $TRAVIS_BRANCH -- test/)
```

Comment:
That sounds like a good shortcut for typing, but for a script, I would prefer to be explicit.